scripts/fdbshow

```diff
@@ -75,8 +75,11 @@ class FdbShow(object):
                 continue
             port_id = self.if_br_oid_map[br_port_id]
             if_name = self.if_oid_map[port_id]
-
-            self.bridge_mac_list.append((int(fdb["vlan"]),) + (fdb["mac"],) + (if_name,))
+            if 'vlan' in fdb:
+                vlan_id = fdb["vlan"]
+            elif 'bvid' in fdb:
+                vlan_id = port_util.get_vlan_id_from_bvid(self.db, fdb["bvid"])
```

Comment:
>vlan_id [](start = 16, length = 7)

If neither exists, vlan_id will be not defined, do you need protection?

Reply:
no actually for lag interfaces there are no if_oid_map in this map,  self.if_oid_map[port_id]. that cause it bails out